src/status_im/protocol/handlers.cljs

```diff
@@ -92,46 +92,52 @@
         (assoc db :sync-listening-started true))
       db)))
 
+(defn- is-debug-message? [{:keys [payload from to]}]
+  (and (= to from)
+       (= (:type payload) :debug)))
+
 (register-handler :incoming-message
   (u/side-effect!
     (fn [_ [_ type {:keys [payload ttl id] :as message}]]
-      (let [message-id (or id (:message-id payload))]
-        (when-not (cache/exists? message-id type)
-          (let [ttl-s             (* 1000 (or ttl 120))
-                processed-message {:id         (random/id)
-                                   :message-id message-id
-                                   :type       type
-                                   :ttl        (+ (dt/now-ms) ttl-s)}]
-            (cache/add! processed-message)
-            (processed-messages/save processed-message))
-          (case type
-            :message (dispatch [:received-protocol-message! message])
-            :group-message (dispatch [:received-protocol-message! message])
-            :ack (if (#{:message :group-message} (:type payload))
-                   (dispatch [:message-delivered message])
-                   (dispatch [:pending-message-remove message]))
-            :seen (dispatch [:message-seen message])
-            :clock-value-request (dispatch [:message-clock-value-request message])
-            :clock-value (dispatch [:message-clock-value message])
-            :group-invitation (dispatch [:group-chat-invite-received message])
-            :update-group (dispatch [:update-group-message message])
-            :add-group-identity (dispatch [:participant-invited-to-group message])
-            :remove-group-identity (dispatch [:participant-removed-from-group message])
-            :leave-group (dispatch [:participant-left-group message])
-            :contact-request (dispatch [:contact-request-received message])
-            :discover (dispatch [:status-received message])
-            :discoveries-request (dispatch [:discoveries-request-received message])
-            :discoveries-response (dispatch [:discoveries-response-received message])
-            :profile (dispatch [:contact-update-received message])
-            :update-keys (dispatch [:update-keys-received message])
-            :online (dispatch [:contact-online-received message])
-            :pending (dispatch [:pending-message-upsert message])
-            :sent (let [{:keys [to id group-id]} message
-                        message' {:from    to
-                                  :payload {:message-id id
-                                            :group-id   group-id}}]
-                    (dispatch [:message-sent message']))
-            (debug "Unknown message type" type)))))))
+      (let [message-id (or id (:message- id payload))]
+        (if (is-debug-message? message)
```

Comment:
we already have this check here https://github.com/status-im/status-react/blob/08a48b3a0de8c33a81db87a72c71fa7143e14676/src/status_im/protocol/handlers.cljs#L113-L113